lib/basket.rb

```diff
@@ -63,4 +70,4 @@ def find_item(product)
   def find_item_index(product)
     @items.index { |i| i.product == product }
```

Comment:
i can try to use `find_index` method -> http://ruby-doc.org/core-2.3.1/Array.html#method-i-find_index